modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryServiceImpl.java

```diff
@@ -30,4 +35,19 @@
 	service = AopService.class
 )
 public class AccountEntryServiceImpl extends AccountEntryServiceBaseImpl {
+
+	@Override
+	public AccountEntry addAccountEntry(
+			long parentAccountEntryId, String name, String description,
+			long logoId, int status, ServiceContext serviceContext)
```

Comment:
Obviously if you implement my suggestion to re-order the method parameters in the local service, we should re-order them here too :)